### PR TITLE
SE-1515

### PR DESCRIPTION
**Pull request by ws-mps**

### CODE REVIEW

File: addons/edi/models/edi_issues.py [hunk 1]
```diff
@@ -35,6 +35,127 @@ class ProjectTask(models.Model):
     )
 
 
+class BaseIssue(models.AbstractModel):
```
Comment (armandcela) [inline addons/edi/models/edi_issues.py:38]: Do we need black format?

File: addons/edi/models/edi_issues.py [hunk 2]
```diff
@@ -35,6 +35,127 @@ class ProjectTask(models.Model):
...
+    # on project module in the future. So if we want to replicate this solution not to depend on
+    # EDI module by force.
+    _name = "base.issues"
+    _inherit = "mail.thread"
+    _description = "Base Issue Object"
+
+    # Empty list variable, will be overriden in the specific models where used.
+    ISSUES_FIELD_MAP = []
+
+    issue_ids = fields.One2many(
```
Comment (armandcela) [inline addons/edi/models/edi_issues.py:51]: Maybe a comment to add the inverse name wherever issue_ids will be used in models.Model classes

File: addons/edi/models/edi_issues.py (hunk 1 above)
Comment (ws-mps) [reply addons/edi/models/edi_issues.py:38]: Not needed for this one.

File: addons/edi/models/edi_issues.py (hunk 2 above)
Comment (ws-mps) [reply addons/edi/models/edi_issues.py:51]: There is no need, thanks.

**Pull request merged**